lib/assert.js

```diff
@@ -89,7 +56,8 @@ function fail(actual, expected, message, operator, stackStartFunction) {
     message = actual;
   if (arguments.length === 2)
     operator = '!=';
-  throw new AssertionError({
+  const errors = lazyErrors();
+  throw new errors.AssertionError({
```

Comment:
Nit: This is anyway `export`ed. Why not just `throw new assert. AssertionError({...});`